engine/src/flutter/docs/contributing/Compiling-the-engine.md

```diff
@@ -16,6 +16,8 @@ Depending on the platform you are making changes for, you may be interested in a
 
 ## General Compilation Tips
 
+- Consider using the `et`, the engine tool, located at
```

Comment:
Stronger language than "consider" maybe? I believe we want this to be the well-lit path.